.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,36 @@
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+| Name| about |title|labels|
+| ------------- | ------------- |------------- |------------- |
+| Feature Request  | Suggest an idea for this project  |[Short Description] (Version)|enhancement|
```

Comment:
Can you add `AYS` to `Version` so that someone can easily define which version should the feature be requested?